src/components/pages/dashboard/panels/map/mapPanel.js

```diff
@@ -14,73 +15,133 @@ import {
 
 import './mapPanel.css';
 
-const atlas = window.atlas;
-const deviceLayer = 'devices-layer';
+const AzureMaps = window.atlas;
```

Comment:
azureMaps?

Reply:
I'm using this as an alias instead of the more generic "atlas"

Reply:
That will be confusing for the new developers to follow. Do we have linting that can validate against these kind of requirements?

Reply:
The idea is that it should be less confusing. Atlas doesn't really tell you what the object is. AzureMaps does. How would linting help here?